### PR TITLE
[Windows] Docker, .NET Core and NPM improvements

### PR DESCRIPTION
**Pull request by maxim-lobanov**
- Merge `Install-Docker.ps1` and `Update-DockerImages.ps1` into the single one to simplify logic. They both are pretty small
- Create separate toolset toggle for .NET Core warmup. Enabled on both Win16 and Win19
- Move the list of npm packages into toolset
- Remove the separate script for `TypeScript` installation

### CODE REVIEW

**Review by AlenaSviridenko — DISMISSED** [review]

**Review by al-cheb — APPROVED** [review]

File: images/win/scripts/Installers/Install-Docker.ps1 [hunk 1]
```diff
@@ -12,4 +12,15 @@ Start-Service docker
 Write-Host "Install-Package Docker-Compose"
 Choco-Install -PackageName docker-compose
 
-Invoke-PesterTests -TestFile "Docker" -TestName "Docker"
\ No newline at end of file
+$dockerToolset = (Get-ToolsetContent).docker
+foreach($dockerImage in $dockerToolset.images) {
```
Comment (miketimofeev) [inline images/win/scripts/Installers/Install-Docker.ps1:16]: Maybe?
```suggestion
$dockerImages = (Get-ToolsetContent).docker.images
foreach($dockerImage in $dockerImages) {
```

File: images/win/scripts/Installers/Install-NodeLts.ps1 [hunk 2]
```diff
@@ -21,14 +21,9 @@ $env:npm_config_prefix = $PrefixPath
...
-npm install -g grunt-cli
-npm install -g gulp-cli
-npm install -g parcel-bundler
-npm install -g --save-dev webpack webpack-cli
-npm install -g yarn
-npm install -g lerna
-npm install -g node-sass
-npm install -g newman
+$npmToolset = (Get-ToolsetContent).npm
+$npmToolset.global_packages | ForEach-Object {
```
Comment (miketimofeev) [inline images/win/scripts/Installers/Install-NodeLts.ps1:25]:
```suggestion
$npmToolset = (Get-ToolsetContent).npm.global_packages
$npmToolset | ForEach-Object {
```

File: images/win/scripts/Tests/Node.Tests.ps1 [hunk 3]
```diff
@@ -7,15 +7,13 @@ Describe "Node.JS" {
         "$ToolName --version" | Should -ReturnZeroExitCode
         }
     }
+
+    $npmToolset = (Get-ToolsetContent).npm
+    $globalNpmPackages = $npmToolset.global_packages | Where-Object { $_.test } | ForEach-Object { $_.test }
```
Comment (miketimofeev) [inline images/win/scripts/Tests/Node.Tests.ps1:12]:
```suggestion
    $npmToolset = (Get-ToolsetContent).npm.global_packages
    $globalNpmPackages = $npmToolset | Where-Object { $_.test } | ForEach-Object { $_.test }
```

**Review by MaksimZhukov — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Review by miketimofeev — DISMISSED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by dmitry-shibanov — APPROVED** [review]

**Review by MaksimZhukov — APPROVED** [review]

**Pull request merged**